### PR TITLE
Finished the changes.

### PR DESCRIPTION
**Pull request by MiqayelChaloyan**
### Changes have been made to several pages.

  * I changed the layout of the profile screen, separated it into several components.
  * I made some more changes in the image screen.

  * After that, I fixed all the modals.

- [ ] Page Screen is more understandable and readable.

### CODE REVIEW

File: screens/ImagesScreen/RenderImagePairs.jsx [hunk 1]
```diff
@@ -17,17 +17,29 @@ const RenderImagePairs = changeProfileImage => {
   const {arrayImages} = useContext(GlobalDataContext);
   const maximum = 4;
   const minimum = 1;
+  const [length1, setLength1] = useState(1);
+  const [length2, setLength2] = useState(4);
```
Comment (M7Arman) [inline screens/ImagesScreen/RenderImagePairs.jsx:21]: length of what? what maximum and minimum? try to choose better names

File: screens/ProfileScreen/profile/HeaderBar.jsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  return (
+    <View style={styles.header}>
+      <Header
+        screen={'Profile'}
+        navigation={navigation}
+        back={'SettingsNav'}
+        continueTo={'LogIn'}
+        root={'Auth'}
+        left={'Settings'}
+        right={'Logout'}
```
Comment (M7Arman) [inline screens/ProfileScreen/profile/HeaderBar.jsx:26]: why inside { }

File: screens/ProfileScreen/profile/HeaderBar.jsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        headerTextColor={theme.colors.primary_white}
+      />
+      <View style={styles.profileImage}>
+        <View>
+          {imageUrl ? (
+            <View>
```
Comment (M7Arman) [inline screens/ProfileScreen/profile/HeaderBar.jsx:32]: why you need this View

File: screens/ProfileScreen/profile/HeaderBar.jsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+              <Image style={styles.userImage} source={{uri: imageUrl}} />
+            </View>
+          ) : (
+            <View>
```
Comment (M7Arman) [inline screens/ProfileScreen/profile/HeaderBar.jsx:36]: why you need this View ?

File: screens/ProfileScreen/profile/index.js [hunk 5]
```diff
@@ -62,109 +54,51 @@ const Profile = ({navigation}) => {
...
-    if (isAnswer === 'PHONE') {
-      accessCamera();
-    } else if (isAnswer === 'STORAGE') {
-      navigation.navigate('Images');
-    }
-    return () => setAnswer('');
+    const handleAnswerChange = () => {
+      setModalVisible(false);
+
+      if (isAnswer === 'PHONE') {
```
Comment (M7Arman) [inline screens/ProfileScreen/profile/index.js:68]: when I see isAnswer I understand it as a boolean variable, but it turned out that it's a string. This is not ok

File: screens/ImagesScreen/RenderImagePairs.jsx (hunk 1 above)
Comment (MiqayelChaloyan) [reply screens/ImagesScreen/RenderImagePairs.jsx:21]: I will fix it.

File: screens/ProfileScreen/profile/HeaderBar.jsx (hunk 2 above)
Comment (MiqayelChaloyan) [reply screens/ProfileScreen/profile/HeaderBar.jsx:26]: I will fix it.

File: screens/ProfileScreen/profile/HeaderBar.jsx (hunk 3 above)
Comment (MiqayelChaloyan) [reply screens/ProfileScreen/profile/HeaderBar.jsx:32]: I will remove unnecessary View.

File: screens/ProfileScreen/profile/HeaderBar.jsx (hunk 4 above)
Comment (MiqayelChaloyan) [reply screens/ProfileScreen/profile/HeaderBar.jsx:36]: I will fix it.

File: screens/ProfileScreen/profile/index.js (hunk 5 above)
Comment (MiqayelChaloyan) [reply screens/ProfileScreen/profile/index.js:68]: I will think of a better solution.

File: screens/ImagesScreen/RenderImagePairs.jsx [hunk 6]
```diff
@@ -17,17 +17,29 @@ const RenderImagePairs = changeProfileImage => {
   const {arrayImages} = useContext(GlobalDataContext);
   const maximum = 4;
   const minimum = 1;
+  const [minimumImageSize, setMinimumImageSize] = useState(1);
+  const [maximumImageSize, setMaximumImageSize] = useState(4);
```
Comment (M7Arman) [inline screens/ImagesScreen/RenderImagePairs.jsx:21]: what are minimum and maximum above these states? if they are constant move out of this component, for naming use THIS_STYLE and also, you can use them in these states defaults

**Pull request merged**